clients/pkg/promtail/promtail_test.go

```diff
@@ -122,6 +122,10 @@ func TestPromtail(t *testing.T) {
 	}()
 	defer p.Shutdown() // In case the test fails before the call to Shutdown below.
 
+	svr := p.server.(*pserver.PromtailServer)
```

Comment:
`PromtailServer` is used here. @MichelHollands

Reply:
Sorry, my mistake. I thought the test was in the same package.

Reply:
Np. thanks for the review :)